drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -69,6 +69,12 @@ local WIND_MODE_MAP = {
   [1]		= capabilities.windMode.windMode.naturalWind
 }
 
+local ROCK_MODE_MAP = {
+  [0] = capabilities.fanOscillationMode.fanOscillationMode.horizontal,
```

Comment:
I think this would be cleaner if this index should be the actual bitmask that corresponds to the given rock mode, that way you can use a value that's defined in our lua libs. Plus, that removes the need to do the bit shifting check below in the rock support functions.

Reply:
I also see this is how it is done elsewhere in the file, so it might be a general improvement we can make outside of the Matter 1.3 changes as well. We can do that in a separate PR and just keep this consistent

Reply:
I made a ticket describing your proposal [here](https://smartthings.atlassian.net/browse/CHAD-13925)